custom/ilsgateway/tanzania/warehouse/updater.py

```diff
@@ -331,9 +244,13 @@ def populate_report_data(start_date, end_date, domain, runner):
         res.get()
 
     non_facilities_chunked_list = chunked(non_facilities, 50)
+
     # then populate everything above a facility off a warehouse table
     for chunk in non_facilities_chunked_list:
-        res = chain(process_non_facility_warehouse_data.si(org, start_date, end_date, runner) for org in chunk)()
+        res = chain(
+            process_non_facility_warehouse_data.si(org, start_date, end_date, runner, strict)
+            for org in chunk
+        )()
```

Comment:
Just curious: Is there a reason we chain these (sequentially) instead of grouping them (in parallel)? i.e. 
```
    res = group(
        process_non_facility_warehouse_data.si(org, start_date, end_date, runner, strict)
        for org in chunk
    )()
```

And could we chain the groups, or stick them all in a single group/chain?, i.e.
```
    res = chain(
        group(
            process_non_facility_warehouse_data.si(org, start_date, end_date, runner, strict)
            for org in chunk
        )()
        for chunk in non_facilities_chunked_list
    )()
    res.get()
```
(Not that it would make a significant performance improvement if we could, because it's all happening in the background queue. Just trying to figure out why we do it the way we do.)